liblwgeom/lwgeom_geos.c

```diff
@@ -1233,6 +1244,83 @@ lwgeom_offsetcurve(const LWLINE* lwline, double size, int quadsegs, int joinStyl
 	return result;
 }
 
+static LWGEOM*
+lwgeom_offsetcurve_multilinestring(const LWGEOM* geom, double size, int quadsegs, int joinStyle, double mitreLimit)
+{
+	int32_t srid = get_result_srid(geom, NULL, __func__);
+	uint8_t is3d = FLAGS_GET_Z(geom->flags);
+	if (srid == SRID_INVALID) return NULL;
+
+	if (!lwgeom_is_collection(geom))
+		return lwgeom_offsetcurve_line(lwgeom_as_lwline(geom), size, quadsegs, joinStyle, mitreLimit);
+	else
+	{
+		LWCOLLECTION* result;
+		LWGEOM* tmp;
+		const LWCOLLECTION* col = lwgeom_as_lwcollection(geom);
+		int i;
+		result = lwcollection_construct_empty(MULTILINETYPE, srid, is3d, LW_FALSE);
+		for (i = 0; i < col->ngeoms; i++)
```

Comment:
`ngeoms` is unsigned, right?

Reply:
@Algunenano how do you get the integer warnings? I miss them on travis :)

Reply:
I have a pretty big list of flags, but you are probably looking for `-Wconversion`. Right now it leads to a lot of warnings, but hopefully I'll address most of them in https://github.com/postgis/postgis/pull/208

Reply:
@dbaston fixed